adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -28,21 +39,40 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 // MakeRequests create bid request for mobfoxpb demand
 func (a *adapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 	var errs []error
-	var err error
 	var tagID string
-
+	var err_tag error
+	var key string
+	var err_key error
+	var route string
+	var method string
 	var adapterRequests []*adapters.RequestData
 
+	requestURI := a.URI
 	reqCopy := *request
 	imp := request.Imp[0]
-	tagID, err = jsonparser.GetString(imp.Ext, "bidder", "TagID")
-	if err != nil {
-		errs = append(errs, err)
+	tagID, err_tag = jsonparser.GetString(imp.Ext, "bidder", "TagID")
```

Comment:
**Nitpick**: Please consider using the variable declare and define shorthand `:=` here rather than defining the variables explicitly above. Meaning, you can change these lines to be:
```
tagID, err_tag := jsonparser.GetString(imp.Ext, "bidder", "TagID")
key, err_key := jsonparser.GetString(imp.Ext, "bidder", "key")
```
And then remove the `err_tag`, `err_key` and `key` variable declarations from L43-45.

Reply:
Removed and renamed to follow single format